maps/site404/site404_announcements.dm

```diff
@@ -0,0 +1,37 @@
+/datum/map/site53
+	emergency_shuttle_docked_message = "Attention all personnel: The emergency shuttle has arrived. You have %ETD% before the shuttle departs."
+	emergency_shuttle_leaving_dock = "Attention all personnel: The emergency shuttle has left the facility, and will be arriving at off-site rendezvous point alpha in %ETA%."
+
+	emergency_shuttle_called_message = "Attention all personnel: Code Gamma initiated, emergency evacuation procedures are now in effect. The emergency shuttle will arrive in %ETA%"
```

Comment:
Semi-Related to your project. You may as well gut the dock, leave, call, and recall messages from this datum, as we handle it in the map defines file anyway. Or gut it from the Map defines and handle it here, either way it should be standardized.

Reply:
Done, does that look right?

Reply:
Looks good.